bee-identity/src/identity.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Holds the `Identity` type.
+
+use std::{fmt, path::Path};
+
+use libp2p_core::{identity::PublicKey, PeerId};
+
+use crate::{migration, pem_file, Keypair as Ed25519Keypair, PublicKey as Ed25519PublicKey};
+
+/// Represents a node identity.
+#[derive(Clone, Debug)]
+pub struct Identity {
+    keypair: Ed25519Keypair,
+    is_fresh: bool,
+}
+
+impl Identity {
+    /// Creates a new identity.
+    pub fn new() -> Self {
+        Self::default()
```

Comment:
isn't in this case ` is_fresh: bool` defaulting to `false` but it should be `true`?

Reply:
have a look at the `default` implementation of `Identity` :trollface: